File: Python/771. Jewels and Stones.py
'''
771. Jewels and Stones
Problem Link : https://leetcode.com/problems/jewels-and-stones/submissions/
Runtime: 16 ms
Memory Usage: 13.6 MB
'''
class Solution(object):
    def numJewelsInStones(self, jewels, stones):
        j=list(jewels)
        s=list(stones)
        c=0
        for x in range(len(s)):
            for y in range(len(j)):
                if s[x]==j[y]:
                    c+=1
        return c

File: java/771 Jewels and Stones.java
/*
771. Jewels and Stones
**********
Problem Link : https://leetcode.com/problems/jewels-and-stones/
Runtime: 1 ms
Memory Usage: 41.6 MB
**********
*/
class Solution {
    public int numJewelsInStones(String J, String S) {
    int jewelcount=0;
     for(int a=0;a<S.length();a++)
     {
         Character stone=S.charAt(a);
         int hashs=stone.hashCode();
         for(int b=0;b<J.length();b++)
         {
            Character jewel=J.charAt(b);
            int hashj=jewel.hashCode();
             if(hashs==hashj)
             {
                 jewelcount++;
             }
         }
     }
        return jewelcount;
    }
}
